run-tests.sh

```diff
@@ -6,9 +6,39 @@
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+export REANA_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://postgres:mysecretpassword@localhost/postgres
+
+# Verify that db container is running before continuing
+check_ready() {
+    RETRIES=40
+    while ! $2
+    do
+        echo "Waiting for $1, $((RETRIES--)) remaining attempts..."
```

Comment:
We either need to change shell from `sh` to `bash`, or else amend POSIX compliance:

```
$ shellcheck ./run-tests.sh 

In ./run-tests.sh line 16:
        echo "Waiting for $1, $((RETRIES--)) remaining attempts..."
                                        ^-- SC2039: In POSIX sh, -- is undefined.


In ./run-tests.sh line 26:
_db_check(){ docker exec --user postgres reana-postgres bash -c "pg_isready" &>/dev/null; }
                                                                             ^---------^ SC2039: In POSIX sh, &> is undefined.

For more information:
  https://www.shellcheck.net/wiki/SC2039 -- In POSIX sh, &> is undefined.

```console